language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -386,83 +399,102 @@ callProcessSilent cmd args = do
+        , "    \"noImplicitReturns\": true,"
+        , "    \"noFallthroughCasesInSwitch\": true,"
+        , "    \"outDir\": \"lib\","
+        , "    \"module\": \"commonjs\","
+        , "    \"declaration\": true,"
+        , "    \"sourceMap\": true"
         , "    },"
-        , "    \"plugins\": ["
-        , "      \"@typescript-eslint\""
-        , "    ],"
-        , "    \"rules\": {"
-        , "      \"@typescript-eslint/no-floating-promises\": \"error\","
-        , "      \"@typescript-eslint/explicit-function-return-type\": \"off\","
-        , "      \"@typescript-eslint/no-inferrable-types\": \"off\""
-        , "    }"
-        , "  },"
-        , "  \"devDependencies\": {"
-        , "    \"@typescript-eslint/eslint-plugin\": \"^2.11.0\","
-        , "    \"@typescript-eslint/parser\": \"^2.11.0\","
-        , "    \"eslint\": \"^6.7.2\","
-        , "    \"typescript\": \"~3.7.3\""
-        , "  }"
+        , "  \"include\": [\"src/**/*.ts\"],"
         , "}"
         ]
 
-writeRootPackageJson :: FilePath -> String -> IO ()
-writeRootPackageJson rootDir workspace =
-   writeFileUTF8 (rootDir </> "package.json") $ unlines
-     [ "{"
-     , "  \"private\": true,"
-     , "  \"workspaces\": ["
-     , "    \"" <> workspace <> "\""
-     , "  ]"
-     , "}"
-     ]
+    writeEsLintConfig :: FilePath -> IO ()
+    writeEsLintConfig dir = writeFileUTF8 (dir </> ".eslintrc.json") $ unlines
+      [ "{"
+      , "  \"parser\": \"@typescript-eslint/parser\","
+      , "  \"parserOptions\": {"
+      , "    \"project\": \"./tsconfig.json\""
+      , "  },"
+      , "  \"plugins\": ["
+      , "    \"@typescript-eslint\""
+      , "  ],"
+      , "  \"extends\": ["
+      , "    \"eslint:recommended\","
+      , "    \"plugin:@typescript-eslint/eslint-recommended\","
+      , "    \"plugin:@typescript-eslint/recommended\","
+      , "    \"plugin:@typescript-eslint/recommended-requiring-type-checking\""
+      , "  ],"
+      , "  \"rules\": {"
+      , "    \"@typescript-eslint/explicit-function-return-type\": \"off\","
+      , "    \"@typescript-eslint/no-inferrable-types\": \"off\""
+      , "  }"
+      , "}"
+      ]
+
+    writePackageJson :: FilePath -> IO ()
+    writePackageJson dir = let name = takeFileName dir in writeFileUTF8 (dir </> "package.json") $ unlines
+            ["{"
+            , "  \"private\": true,"
+            , "  \"name\": \"@daml2ts/" <> name <> "\","
+            , "  \"version\": \"" <> sdkVersion <> "\","
+            , "  \"description\": \"Produced by daml2ts\","
+            , "  \"license\": \"Apache-2.0\","
+            , "  \"dependencies\": {"
+            , "    \"@daml/types\": \"" <> sdkVersion <> "\","
+            , "    \"@mojotech/json-type-validation\": \"^3.1.0\""
+            , "  },"
+            , "  \"scripts\": {"
+            , "    \"build\": \"tsc --build\","
+            , "    \"lint\": \"eslint --ext .js,.ts src/ --max-warnings 0\""
```

Comment:
```suggestion
            , "    \"lint\": \"eslint --ext .ts src/ --max-warnings 0\""
```
We don't want to lint the `.js` files. We might need to call `yarn lint` before `yarn build` in order to avoid `yarn lint` lint falling over the `.js` files in `lib/`.

Reply:
Removeing the `.js`.